app/src/Service/Ldap/Client.php

```diff
@@ -143,23 +144,28 @@ public function create(string $fullDn, array $attributes): bool
     }
 
     /**
+     * Finding and updating an existing entry
+     *
      * @return bool
      *
      * @throws LdapException
      */
     public function update(string $fullDn, string $query, array $attributes = []) : bool
     {
-        // Finding and updating an existing entry
         $entryManager = $this->ldap->getEntryManager();
         $entry = $this->get($query, $fullDn);
 
         if (empty($entry)) {
             return false;
         }
 
+        foreach ($entry->getAttributes() as $key => $value) {
+            $entry->setAttribute($key, []);
+        }
         foreach ($attributes as $key => $value) {
-            $entry->setAttribute($key, $value);
+            $entry->setAttribute($key, $attributes[$key]);
```

Comment:
What ??? We already have the value from the foreach loop, so why reference from array again ?! 
```suggestion
            $entry->setAttribute($key, $value);
```